DigitalLearningSolutions.Web/Views/Shared/Components/DateRangeInput/_DateInput.cshtml

```diff
@@ -0,0 +1,55 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Common.ViewComponents
+@model DateRangeInputDateInputViewModel
+
+@{
+  var dayErrorCss = Model.HasDayError ? "nhsuk-input--error" : "";
+  var monthErrorCss = Model.HasMonthError ? "nhsuk-input--error" : "";
+  var yearErrorCss = Model.HasYearError ? "nhsuk-input--error" : "";
+  var conditionalInputClass = Model.IsConditionalDateInput ? "nhsuk-checkboxes__conditional" : "";
+  var conditionalInputId = Model.IsConditionalDateInput ? "conditional-end-date" : "start-date";

```

Comment:
What's this for? Why is the dateinput component concerned about end & start dates?

Reply:
This isn't a DateInput Component, this is a DateInput partial as part of a DateRange component.